third_party/gpus/crosstool/clang/bin/crosstool_wrapper_driver_rocm.tpl

```diff
@@ -189,8 +191,11 @@ def InvokeHipcc(argv, log=False):
 
   # TODO(zhengxq): for some reason, 'gcc' needs this help to find 'as'.
   # Need to investigate and fix.
-  cmd = 'PATH=' + PREFIX_DIR + ':$PATH ' + cmd
+  cmd = 'PATH=' + PREFIX_DIR + ':$PATH '\
+        + HIPCC_ENV + ' '\
```

Comment:
@yxsamliu could you explain the possible values for `HIPCC_ENV`. specifically I want to understand the difference between `hip/hcc` and `hip-clang` toolchains here

Reply:
Currently only environment variable HIP_CLANG_PATH, DEVICE_LIB_PATH, HIP_VDI_HOME, and HIPCC_VERBOSE are passed to HIPCC_ENV.

If either HIP_CLANG_PATH or HIP_VDI_HOME is set, hipcc will use hip-clang to compile/link HIP programs instead of hcc. hipcc will choose suitable options, include paths, and library path for hip-clang.

This allows user to choose hip-clang to build TensorFlow by setting environment variable.